test/fulfillmentSpec.js

```diff
@@ -332,162 +326,4 @@ describe('PUT /fulfillment', function () {
     expect((yield getAccount('alice')).balance).to.equal(100)
     expect((yield getAccount('bob')).balance).to.equal(0)
   })
-
-  it('should trigger subscriptions when notification is executed', function * () {
```

Comment:
Line 395 of this file tests that the fulfillment was attached with the notification. Could you add a test in the `accountSpec.js` that tests for that as well? Also, if you don't think it makes sense to have those tests in that file (I'd definitely agree), you could move them to a more sensible place